httpobsdashboard/conf/deviations.json

```diff
@@ -180,5 +180,42 @@
     "tlsobs": {
       "pass": true
     }
+  },
+  "HTTP Only": {
```

Comment:
I have no problem with this as a deviation, but I think it would be better to use something other than `HTTP Only`. All of these sites (except pypi for some reason) _do_ support HTTPS, they just don't forcibly redirect. Something like `No Redirection` or along those lines would be fine.